seed/migrations/0102_auto_20190415_1115.py

```diff
@@ -0,0 +1,119 @@
+# -*- coding: utf-8 -*-
```

Comment:
@nllong Thanks for combining the two migrations! I ran into errors trying to let Django create this combined migration (letting Django read the models), so I assumed the two migrations I ended up creating needed to be kept separate. Travis test passed so it turns out the migrations can be combined. Thanks for catching this! 😄